src/Rules/Exclude.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace StellarWP\Validation\Rules;
+
+use Closure;
+use StellarWP\Validation\Commands\ExcludeValue;
+use StellarWP\Validation\Contracts\ValidationRule;
+
+/**
+ * Applying this rule will prevent all further validations and exclude the value from the validated dataset.
+ *
+ * @unreleased
+ */
+class Exclude implements ValidationRule
+{
+    /**
+     * @inheritDoc
+     *
+     * @unreleased
+     */
+    public static function id(): string
+    {
+        return 'exclude';
+    }
+
+    /**
+     * @inheritDoc
+     *
+     * @unreleased
+     */
+    public static function fromString(string $options = null): ValidationRule
+    {
+        return new self();
```

Comment:
Do we want to use `new static()` in case we want to be able to extend `Exclude`?

Reply:
I've become very reluctant to use `new static()` prematurely, as it's bitten me in the past. It's not obvious, but when using `new static()` one is locking in the constructor for child classes, so to pass static analysis one must use an interface or make `__construct()` final. Either way, you end up limiting inheritance. So until a use case comes up, I've found it's better to assume child classes will override this method if need be, allowing them to also change the constructor.